pkg/kubectl/cmd/auth/cani.go

```diff
@@ -393,3 +395,13 @@ func isNamespaced(gvr schema.GroupVersionResource, discoveryClient discovery.Dis
 
 	return false, fmt.Errorf("the server doesn't have a resource type '%s' in group '%s'", gvr.Resource, gvr.Group)
 }
+
+func isVerb(s string) error {
+	verbs := []string{"create", "watch", "post", "put", "list", "get", "delete", "deletecollection", "update", "patch"}
```

Comment:
suggest one check/warning for nonResourceURL checks: get, put, post, head, options, delete, patch

and another check/warning for resource verbs:
* standard crud resource verbs (get, list, watch, create, update, patch, delete, deletecollection)
* known special verbs: use, bind, escalate, `*`